package/CHANGELOG

```diff
@@ -61,6 +61,10 @@ Enhancements
     analysis.nucleicacids
   * CRD extended format can now be explicitly requested with the `extended`
     keyword (Issue #3605)
+  * Added benchmarks in lib.distances for the following functions .
```

Comment:
Can you make this a fewer lines? Other than that LGTM.

Reply:
@hmacdope i am working on it.